vite.config.ts

```diff
@@ -2,17 +2,24 @@ import { defineConfig } from '@ownclouders/extension-sdk'
 import { readFileSync } from 'fs'
 import { join } from 'path'
 
-const certsDir = `${__dirname}/dev/docker/traefik/certificates`
+const isProd = process.env.PROD
+
+const server = isProd
+  ? false
+  : (() => {
+      const certsDir = `${__dirname}/dev/docker/traefik/certificates`
+      return {
+        port: 9999,
+        host: 'host.docker.internal',
+        https: {
+          key: readFileSync(join(certsDir, 'server.key')),
+          cert: readFileSync(join(certsDir, 'server.crt'))
+        }
+      }
+    })()
```

Comment:
```suggestion
const isProd = process.env.PROD

const certsDir = `${__dirname}/dev/docker/traefik/certificates`
const serverConfig = {
        port: 9999,
        host: 'host.docker.internal',
        https: {
          key: readFileSync(join(certsDir, 'server.key')),
          cert: readFileSync(join(certsDir, 'server.crt'))
        }
      }
```

Reply:
`const certsDir = `${__dirname}/dev/docker/traefik/certificates`` This leads to failure because it can't find the file on production build and readFileSync will throw error.